lib/options.nix

```diff
@@ -116,6 +114,8 @@ rec {
 
      You can omit the default path if the name of the option is also attribute path in nixpkgs.
 
+     Type: mkPackageOption :: pkgs -> string -> { default :: [string], example :: null | string | [string] } -> option
```

Comment:
moved the type comment. Because nixdoc had trouble parsing it. -> easiest fix right now.  I will improve nixdoc later.